sqldb/config.go

```diff
@@ -0,0 +1,55 @@
+package sqldb
+
+import (
+	"fmt"
+	"net/url"
+	"time"
+)
+
+const (
+	// defaultMaxConns is the number of permitted active and idle
+	// connections. We want to limit this so it isn't unlimited. We use the
+	// same value for the number of idle connections as, this can speed up
+	// queries given a new connection doesn't need to be established each
+	// time.
+	defaultMaxConns = 25
+
+	// connIdleLifetime is the amount of time a connection can be idle.
+	connIdleLifetime = 5 * time.Minute
+)
+
+// SqliteConfig holds all the config arguments needed to interact with our
+// sqlite DB.
+//
+//nolint:lll
+type SqliteConfig struct {
```

Comment:
Perhaps we combined in this commit during a rebase? (doesn't match commit title)

Reply:
No it's just misleading as I move things around to break dependencies so it compiles on wasm.